build.sbt

```diff
@@ -44,6 +46,40 @@ lazy val toolkitTest = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     mimaPreviousArtifacts := Set()
   )
 
+lazy val toolkitTesting = project
+  .in(file("toolkit-testing"))
+  .settings(
+    name := "toolkit-testing",
+    Test / test := (Test / test).dependsOn(toolkit.jvm / publishLocal).value,
+    scalacOptions ++= {
+      if (scalaBinaryVersion.value == "2.13") Seq("-Ytasty-reader") else Nil
+    },
+    libraryDependencies ++= Seq(
+      "org.typelevel" %% "munit-cats-effect" % "2.0.0-M3" % Test,
+      "co.fs2" %% "fs2-io" % "3.9.2" % Test,
+      // https://github.com/VirtusLab/scala-cli/issues/2421
+      "org.virtuslab.scala-cli" %% "cli" % "1.0.4" % Test cross (CrossVersion.for2_13Use3) excludeAll (
+        ExclusionRule("com.lihaoyi:geny_2.13"),
+        ExclusionRule(
+          "org.scala-lang.modules",
+          "scala-collection-compat_2.13"
+        ),
+        ExclusionRule(
+          "com.github.plokhotnyuk.jsoniter-scala",
+          "jsoniter-scala-core_2.13"
+        ),
+        ExclusionRule("com.lihaoyi", "sourcecode_2.13"),
+        ExclusionRule("ai.kien", "python-native-libs_2.13"),
+        ExclusionRule("com.lihaoyi", "os-lib_2.13")
+      )
+    ),
+    Test / fork := true,
+    Test / javaOptions += s"-Dtoolkit.testing.classpath=${(Test / fullClasspath).value
+        .map(_.data.getAbsolutePath)
+        .mkString(File.pathSeparator)}"
```

Comment:
Instead of this, we can just path the classpath as a `BuildInfo` key. Then we also don't need to hack.

Also `Compile / dependencyClasspath` should be sufficient if we can move scala-cli out of test scope.

Reply:
You could do this only if `Compile / dependencyClasspath` is a setting. You can't depend on tasks, neither using `.value` in settings definition. I'll try but I'm not sure.

Reply:
Ahhh. Well that's annoying, forgot that. Ok, so then maybe we need to pass it as an env variable so that JS and Native can pick it up.

Reply:
😎 
```scala
    buildInfoKeys +=
      BuildInfoKey.map(Compile / dependencyClasspath) { case (k, v) =>
        (k, v.seq.map(_.data.getAbsolutePath).mkString(File.pathSeparator))
      }
```

Reply:
This can be used in the CE build too :)